MLS.Agent.Tests/CommandLine/CommandLineParserTests.cs

```diff
@@ -27,7 +28,7 @@ public class CommandLineParserTests : IDisposable
         private PackOptions _packOptions;
         private InstallOptions _installOptions;
         private PackageSource _installPackageSource;
-        private VerifyOptions _verifyOptions;
```

Comment:
Is the `VerifyOptions` class still being used? Thoughts on changing its `Dir` property to be a `IDirectoryAccessor` and keeping the options pattern?

Reply:
Changed the verify options and the startup options to only use the directory accessor